PyISY/Nodes/__init__.py

```diff
@@ -194,7 +194,10 @@ def _controlmsg(self, xmldoc):
         prec = attr_from_xml(xmldoc, ATTR_ACTION, ATTR_PREC, None)
         uom = attr_from_xml(xmldoc, ATTR_ACTION, ATTR_UOM, None)
 
-        self.get_by_id(nid).controlEvents.notify(EventResult(cntrl, nval, prec, uom))
+        node = self.get_by_id(nid)
+        if not node: return
```

Comment:
Maybe this deserves a DEBUG level log about the unknown ID? Do we know why this happens?

Reply:
Good call.  From what I've seen (and if I remember this one correctly), this usually happens if you add a device or node to the ISY while this is running and you receive a message (control the device) before you restart the module.

Reply:
Should this be a DEBUG or a INFO/WARNING with a suggestion to restart?

Reply:
Let's go with DEBUG for now, and strive to make hot-adding nodes to the Insteon network something we support in the future. At least that's my opinion. I could go either way.

Mostly I can't figure out a good wording for the WARNING that is not home-assistant specific ;)

Reply:
OK. Sounds good. I'll push it here shortly.

It shouldn't be too hard to implement, but would probably be best after moving to asyncio (or similar). I'm thinking if you get a _controlmsg for a node you don't know about, try calling `rest/nodes/--unknown node--` and see if you get a good response. If so, add the node an process.